templates/zerver/reset_confirm.html

```diff
@@ -43,11 +44,12 @@ <h1>{{ _('Set a new password.') }}</h1>
                         </div>
                     </div>
                 </div>
-                <div class="input-box">
+                <div class="input-box password-div">
                     <label for="id_new_password2" class="">{{ _('Confirm password') }}</label>
                     <input id="id_new_password2" class="required" type="password" name="new_password2"
                       value="{% if form.new_password2.value() %}{{ form.new_password2.value() }}{% endif %}"
                       maxlength="100" required />
```

Comment:
This one should have autocomplete=off, right?

Reply:
Yep.